src/interactive-window/interactiveWindowCommandListener.ts

```diff
@@ -241,7 +242,7 @@ export class InteractiveWindowCommandListener implements IDataScienceCommandList
                             openQuestion1
                         );
                         if (selection === openQuestion1) {
-                            await this.ipynbProvider.open(uri);
+                            await openAndShowNotebook(uri);
```

Comment:
Should we remove the `open` method from the `InotebookEditorProvider` interface?